crypto/fipsmodule/evp/evp.c

```diff
@@ -242,6 +242,38 @@ int EVP_PKEY_type(int nid) {
   return meth->pkey_id;
 }
 
+EVP_PKEY *EVP_PKEY_new_mac_key(int type, ENGINE *engine, const uint8_t *mac_key,
```

Comment:
That this thing returns `EVP_PKEY` means we minimally must keep the key in the `EVP_PKEY` object. But meh.